Sources/NIOPosix/Bootstrap.swift

```diff
@@ -2417,8 +2417,8 @@ extension NIOPipeBootstrap {
                 try self.validateFileDescriptorIsNotAFile(output)
             }
 
-            inputFileHandle = input.flatMap { NIOFileHandle(descriptor: $0) }
-            outputFileHandle = output.flatMap { NIOFileHandle(descriptor: $0) }
+            inputFileHandle = input.flatMap { NIOFileHandle(_deprecatedTakingOwnershipOfDescriptor: $0) }
+            outputFileHandle = output.flatMap { NIOFileHandle(_deprecatedTakingOwnershipOfDescriptor: $0) }
```

Comment:
These make it look like we're gonna need a way to have an internal non-deprecated flow that can be used for the few internal use-cases we have that aren't totally FUBAR. And then a GH issue to get us to transition away from it. Alternatively, we need a different spelling here.

Reply:
@Lukasa good catch. Actually, I don't think we need to go via NIOFileHandle at all here.  gimme a sec

Reply:
@Lukasa yes, fixed, gone now